### PR TITLE
Dockerfile - Add new stage for minify js file

### PR DESCRIPTION
**Pull request by vttranlina**
resolve https://github.com/linagora/tmail-flutter/issues/1918

## Before
```
-rw-r--r-- 1 root root 17748811 May 15 02:35 main.dart.js
-rw-r--r-- 1 root root  2687864 May 15 02:35 main.dart.js.gz
-rw-r--r-- 1 root root    59808 May 15 02:35 main.dart.js_1.part.js
-rw-r--r-- 1 root root     9731 May 15 02:35 main.dart.js_1.part.js.gz
-rw-r--r-- 1 root root   315200 May 15 02:35 main.dart.js_10.part.js
-rw-r--r-- 1 root root    42360 May 15 02:35 main.dart.js_10.part.js.gz
-rw-r--r-- 1 root root      870 May 15 02:35 main.dart.js_2.part.js
-rw-r--r-- 1 root root      470 May 15 02:35 main.dart.js_2.part.js.gz
-rw-r--r-- 1 root root     1340 May 15 02:35 main.dart.js_3.part.js
-rw-r--r-- 1 root root      681 May 15 02:35 main.dart.js_3.part.js.gz
-rw-r--r-- 1 root root   689363 May 15 02:35 main.dart.js_4.part.js
-rw-r--r-- 1 root root    93698 May 15 02:35 main.dart.js_4.part.js.gz
-rw-r--r-- 1 root root   113888 May 15 02:35 main.dart.js_5.part.js
-rw-r--r-- 1 root root    21285 May 15 02:35 main.dart.js_5.part.js.gz
-rw-r--r-- 1 root root     1052 May 15 02:35 main.dart.js_6.part.js
-rw-r--r-- 1 root root      545 May 15 02:35 main.dart.js_6.part.js.gz
-rw-r--r-- 1 root root   117658 May 15 02:35 main.dart.js_7.part.js
-rw-r--r-- 1 root root    21452 May 15 02:35 main.dart.js_7.part.js.gz
-rw-r--r-- 1 root root    62665 May 15 02:35 main.dart.js_8.part.js
-rw-r--r-- 1 root root    10672 May 15 02:35 main.dart.js_8.part.js.gz
-rw-r--r-- 1 root root    35221 May 15 02:35 main.dart.js_9.part.js
-rw-r--r-- 1 root root     5999 May 15 02:35 main.dart.js_9.part.js.gz

```
## After
```
-rw-r--r-- 1 root root 11402014 Jun 12 13:13 main.dart.js
-rw-r--r-- 1 root root  2229687 Jun 12 13:13 main.dart.js.gz
-rw-r--r-- 1 root root    41665 Jun 12 13:13 main.dart.js_1.part.js
-rw-r--r-- 1 root root     8352 Jun 12 13:13 main.dart.js_1.part.js.gz
-rw-r--r-- 1 root root   221283 Jun 12 13:13 main.dart.js_10.part.js
-rw-r--r-- 1 root root    36853 Jun 12 13:13 main.dart.js_10.part.js.gz
-rw-r--r-- 1 root root      475 Jun 12 13:13 main.dart.js_2.part.js
-rw-r--r-- 1 root root      263 Jun 12 13:13 main.dart.js_2.part.js.gz
-rw-r--r-- 1 root root      755 Jun 12 13:13 main.dart.js_3.part.js
-rw-r--r-- 1 root root      442 Jun 12 13:13 main.dart.js_3.part.js.gz
-rw-r--r-- 1 root root   476854 Jun 12 13:13 main.dart.js_4.part.js
-rw-r--r-- 1 root root    80465 Jun 12 13:13 main.dart.js_4.part.js.gz
-rw-r--r-- 1 root root    76746 Jun 12 13:13 main.dart.js_5.part.js
-rw-r--r-- 1 root root    17494 Jun 12 13:13 main.dart.js_5.part.js.gz
-rw-r--r-- 1 root root    72627 Jun 12 13:13 main.dart.js_6.part.js
-rw-r--r-- 1 root root    17092 Jun 12 13:13 main.dart.js_6.part.js.gz
-rw-r--r-- 1 root root    41147 Jun 12 13:13 main.dart.js_7.part.js
-rw-r--r-- 1 root root     8588 Jun 12 13:13 main.dart.js_7.part.js.gz
-rw-r--r-- 1 root root    24373 Jun 12 13:13 main.dart.js_8.part.js
-rw-r--r-- 1 root root     5186 Jun 12 13:13 main.dart.js_8.part.js.gz
-rw-r--r-- 1 root root      449 Jun 12 13:13 main.dart.js_9.part.js
-rw-r--r-- 1 root root      271 Jun 12 13:13 main.dart.js_9.part.js.gz
```

### CODE REVIEW

**Comment by tk-nguyen** [comment]
What are you using to minify the js files?

**Comment by vttranlina** [comment]
> What are you using to minify the js files?

[uglifyjs](https://www.npmjs.com/package/uglify-js)

**Comment by vttranlina** [comment]
> Do you happen to know any other minify tools? 

No, I'm not an expert in it, `[uglifyjs](https://www.npmjs.com/package/uglify-js)` is in the top result of Stackoverflow, so I tried it.

File: Dockerfile [hunk 1]
```diff
@@ -10,11 +10,19 @@ COPY . .
 # Precompile tmail flutter
 RUN bash prebuild.sh && flutter build web --profile
```
[the remark this answers is not in the document]
Comment (hoangdat) [reply Dockerfile:11]: https://github.com/linagora/tmail-flutter/commit/5221a24aea4c7c969f2bf0a27aac947d48484b9f IMO, no problem with it :))

**Comment by tk-nguyen** [comment]
Alright, I've done some benchmarks with [Google Lighthouse](https://github.com/GoogleChrome/lighthouse). 

There is no difference in loading time between minifying and not minifying the js files built with `--profile`.

There is a 20% improvement in loading time when building with `--release`.

Here are the Lighthouse benchmarks:
  
[tmail-benchmarks.zip](https://github.com/linagora/tmail-flutter/files/11743780/tmail-benchmarks.zip)

**Review by tk-nguyen — APPROVED** [review]

**Comment by chibenwa** [comment]
@hoangdat you agree with this?

**Pull request merged**